src/Tribe/Template/Month.php

```diff
@@ -394,26 +394,25 @@ protected function set_events_in_month() {
 
 			$post_stati = implode( "','", $post_stati );
 
-			$events_request        =
-				$wpdb->prepare(
-					"SELECT tribe_event_start.post_id as ID,
-							tribe_event_start.meta_value as EventStartDate,
-							tribe_event_end_date.meta_value as EventEndDate
-					FROM $wpdb->postmeta AS tribe_event_start
-					LEFT JOIN $wpdb->posts ON tribe_event_start.post_id = $wpdb->posts.ID
-					LEFT JOIN $wpdb->postmeta as tribe_event_end_date ON ( tribe_event_start.post_id = tribe_event_end_date.post_id AND tribe_event_end_date.meta_key = '_EventEndDate' )
-					WHERE tribe_event_start.meta_key = '_EventStartDate'
-					AND ( (tribe_event_start.meta_value >= '%1\$s' AND  tribe_event_start.meta_value <= '%2\$s')
-						OR (tribe_event_start.meta_value <= '%1\$s' AND tribe_event_end_date.meta_value >= '%1\$s')
-						OR ( tribe_event_start.meta_value >= '%1\$s' AND  tribe_event_start.meta_value <= '%2\$s')
-					)
-					AND $wpdb->posts.post_status IN('$post_stati')
-					ORDER BY $wpdb->posts.menu_order ASC, DATE(tribe_event_start.meta_value) ASC, TIME(tribe_event_start.meta_value) ASC;
-					",
-					$grid_start_datetime,
-					$grid_end_datetime
+			$events_request = $wpdb->prepare(
+				"SELECT tribe_event_start.post_id as ID,
+						tribe_event_start.meta_value as EventStartDate,
+						tribe_event_end_date.meta_value as EventEndDate
+				FROM $wpdb->postmeta AS tribe_event_start
+				LEFT JOIN $wpdb->posts ON tribe_event_start.post_id = $wpdb->posts.ID
+				LEFT JOIN $wpdb->postmeta as tribe_event_end_date ON ( tribe_event_start.post_id = tribe_event_end_date.post_id AND tribe_event_end_date.meta_key = '_EventEndDate' )
+				WHERE tribe_event_start.meta_key = '_EventStartDate'
+				AND ( (tribe_event_start.meta_value >= '%1\$s' AND  tribe_event_start.meta_value <= '%2\$s')
+					OR (tribe_event_start.meta_value <= '%1\$s' AND tribe_event_end_date.meta_value >= '%1\$s')
+					OR ( tribe_event_start.meta_value >= '%1\$s' AND  tribe_event_start.meta_value <= '%2\$s')
+				)
+				AND $wpdb->posts.post_status IN('$post_stati')
+				ORDER BY $wpdb->posts.menu_order ASC, DATE(tribe_event_start.meta_value) ASC, TIME(tribe_event_start.meta_value) ASC;
+				",
+				$grid_start_datetime,
+				$grid_end_datetime
+			);
```

Comment:
Note: this was just a whitespace change on this block.

1. removed extraneous spacing around the equal sign
1. Moved the `$wpdb->prepare` to the same line as the equal
1. Tabbed stuff back by one tab.